src/js/node/fs.promises.ts

```diff
@@ -159,7 +159,13 @@ const exports = {
   close: fs.close.bind(fs),
   copyFile: fs.copyFile.bind(fs),
   cp,
-  exists: fs.exists.bind(fs),
+  exists: async function exists() {
+    try {
+      return fs.exists.$apply(fs, arguments);
```

Comment:
We have to await this value or a rejected Promise will throw

```suggestion
      return await fs.exists.$apply(fs, arguments);
```